react/features/settings/components/native/ProfileView.tsx

```diff
@@ -103,7 +105,9 @@ const ProfileView = ({ isInWelcomePage }: {
     useLayoutEffect(() => {
         navigation.setOptions({
             headerLeft,
-            headerRight: !isInWelcomePage && headerRight
+            headerRight: !isInWelcomePage
+                && !locationURL?.hostname?.includes('8x8.vc')
```

Comment:
This will detect not only jaas. Can we use the isVpass function?

Reply:
Like https://github.com/jitsi/jitsi-meet/commit/3811caa8a00114523c2f18df9c070522d1353d8c

Reply:
It's intentional, same should apply on 8x8.vc. Login on that domain won't work on mobile.

Reply:
Aham, as the commit says jaas. But if it will not work, ok.

Reply:
Right. Calin pl change the commit message when merging.